va/va.go

```diff
@@ -524,3 +540,25 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 
 	return body, url.String(), nil
 }
+
+//this reverseaddr funntion is from net/dnsclient.go form golang
```

Comment:
nit: You're missing a leading space after the `//` here and also have a couple small typos:
``` suggestion
// reverseaddr function is borrowed from net/dnsclient.go[0] and the Go std library.
// [0]: https://golang.org/src/net/dnsclient.go
```